tests/functional/disaster-recovery/regional-dr/test_2az_failover_relocate_zone_failure_test.py

```
@@ -0,0 +1,680 @@
+                )
+
+            logger.info(
+                f"Stopping {len(zone_nodes)} node(s): "
+                f"{[node.name for node in zone_nodes]}"
+            )
+            nodes_multicluster[first_workload["primary_cluster_index"]].stop_nodes(
+                zone_nodes
+            )
+            logger.info(
+                f"Node(s) in zone '{power_off_zone}' stopped successfully. "
+                f"Waiting 10 min for OpenShift to detect the node failure ..."
```

Ruff (0.15.18)

[warning] Logging statement uses f-string

(G004)